### DISCUSSION TITLE
Setup question: horizontal wled strip on side of tv

### DISCUSSION THREAD

**Discussion by Nicko33127** [awawa-dev/HyperHDR#981] [General]
I hope you don't mind me asking for a how to set question, please delete if not allowed.

I have a Philips TV with ambilight already. I am also in the work of adding few esp-wleds around the living room. 4 m strip behind the TV, and currently building strips along the book shelve on each sides of the TV. Initially for room ambience, but as my home assistant pc is in the media cabinet under the TV I added a capture card between the AC receiver and the tv to run hyperhdr. Probably too much light but I want to try.

My question is, how to set up the 5 segments of the light strips if they are all horizontal,but on the side of the tv? So the top shelve get the colour of the top corner of the tv , middle shelves middle of TV ect. Is this even possible?

I would also like to have a button on home assistant that can turn one hyperhdr on and off fully. So it's off by default and the strip are just controlled by wled, but on when I want the added effects.

![20241114_222245](https://github.com/user-attachments/assets/0794cd4b-9d55-4662-a54d-bf908e566eea)

**Comment by awawa-dev** [comment]
Hi
Usually segments are not the problem using multiple-instance in HyperHDR or define them in WLED (I'm not providing support for WLED, I prefer wired solution). I think that here even one instance in HyperHDR would be enough, although you would probably have to set the layout in json mainly manually.
